test/e2e/network/dns_common.go

```diff
@@ -613,3 +613,31 @@ func generateDNSUtilsPod() *v1.Pod {
 		},
 	}
 }
+
+func getDNSDomain(f *framework.Framework) []string {
+
+	createUtilsPod := generateDNSUtilsPod()
+	createUtilsPod, err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(createUtilsPod)
+	Expect(err).NotTo(HaveOccurred(), "failed to create pod: %s", createUtilsPod.Name)
+	framework.Logf("Created pod %v", createUtilsPod)
+	defer func() {
+		framework.Logf("Deleting pod %s...", createUtilsPod.Name)
+		if err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).Delete(createUtilsPod.Name, metav1.NewDeleteOptions(0)); err != nil {
+			framework.Failf("Failed to delete pod %s: %v", createUtilsPod.Name, err)
+		}
+	}()
+	Expect(f.WaitForPodRunning(createUtilsPod.Name)).NotTo(HaveOccurred(), "failed to wait for pod %s to be running", createUtilsPod.Name)
+	cmd := []string{"cat", "/etc/resolv.conf"}
+	stdout, stderr, err := f.ExecWithOptions(framework.ExecOptions{
+		Command:       cmd,
+		Namespace:     f.Namespace.Name,
+		PodName:       createUtilsPod.Name,
+		ContainerName: "util",
+		CaptureStdout: true,
+		CaptureStderr: true,
+	})
+	Expect(err).NotTo(HaveOccurred(), "failed to examine resolv.conf file on pod, stdout: %v, stderr: %v, err: %v", stdout, stderr, err)
+	res := strings.Split(stdout, "\n")
```

Comment:
are you sure of the format of this file ? I mean can you have wrong lenght of arrays ?